From: =?utf-8?q?Guido_G=C3=BCnther?= <agx@sigxcpu.org>
Date: Wed, 25 Mar 2020 10:10:18 +0100
Subject: Disable atk during tests

This fixes test problems on GNU Hurd.

Thanks: Samuel Thibault for investigating
Closes: #953971
---
 tests/meson.build | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tests/meson.build b/tests/meson.build
index 95210ba..53a5e46 100644
--- a/tests/meson.build
+++ b/tests/meson.build
@@ -7,6 +7,7 @@ test_env = [
   'GSETTINGS_BACKEND=memory',
   'PYTHONDONTWRITEBYTECODE=yes',
   'MALLOC_CHECK_=2',
+  'NO_AT_BRIDGE=1',
 ]
 
 test_cflags = [
